### PR TITLE
Ogc 159 link in announcement

### PR DESCRIPTION
**Pull request by BreathingFlesh**
Please fill in the commit message below and work through the checklist. You can delete parts that are not needed, e.g. the optional description, the link to a ticket or irrelevant options of the checklist.

## Commit message

Org: Add option for linking announcement

TYPE: Feature
LINK: OGC-159

## Checklist

- [x] I made changes/features for both org and town6
- [x] I have performed a self-review of my code
- [x] I considered adding a reviewer
- [x] I have tested my code thoroughly by hand
    - [x] I have tested styling changes/features on different browsers
- [x] I have added tests for my changes/features
- [x] I have updated the PO files

### CODE REVIEW

File: src/onegov/org/templates/layout.pt [hunk 1]
```diff
@@ -84,13 +84,16 @@
...
 
         <header tal:define="header_options request.app.org.header_options or {}" >
-          <div tal:condition="header_options.get('left_header_announcement', '')"
-               id="announcement"
-               style="color: ${header_options.get('left_header_announcement_font_color', '#000000')};
-                      background-color: ${header_options.get('left_header_announcement_bg_color', '#FBBC05')};">
-              <div class="grid-container">
-                ${header_options['left_header_announcement']}
-              </div>
+          <div tal:condition="header_options.get('announcement', '') and (request.is_manager or not header_options.get('announcement_is_private'))"
```
Comment (msom) [inline src/onegov/org/templates/layout.pt:87]: Description in the settings is "not logged in" but here it's is_manager?

**Review by msom — CHANGES_REQUESTED** [review]

File: src/onegov/town6/templates/layout.pt [hunk 2]
```diff
@@ -80,12 +80,15 @@
             <a class="show-for-sr" href="#content" i18n:translate>Skip navigation</a>
             <main>
                 <header id="header" tal:define="header_options request.app.org.header_options or {}">
-                    <div tal:condition="header_options.get('left_header_announcement', '')"
-                         id="announcement"
-                         style="color: ${header_options.get('left_header_announcement_font_color', '#000000')};
-                                background-color: ${header_options.get('left_header_announcement_bg_color', '#FBBC05')};">
-                             <div class="grid-container">
-                                    ${header_options['left_header_announcement']}
+                    <div tal:condition="header_options.get('announcement', '') and (request.is_manager or not header_options.get('announcement_is_private'))"
```
[the remark this answers is not in the document]
Comment (BreathingFlesh) [reply src/onegov/town6/templates/layout.pt:83]: But the description is "Only show Announcement for logged-in users"?

File: src/onegov/org/templates/layout.pt (hunk 1 above)
Comment (BreathingFlesh) [reply src/onegov/org/templates/layout.pt:87]: But the description is "Only show Announcement for logged-in users"?

File: src/onegov/town6/templates/layout.pt (hunk 2 above)
[the remark this answers is not in the document]
Comment (msom) [reply src/onegov/town6/templates/layout.pt:83]: logged-in = admin, editor, member; is_manger = admin, editor

**Pull request merged**